src/pages/api/configs/tryPassword.tsx

```diff
@@ -8,6 +9,12 @@ function Post(req: NextApiRequest, res: NextApiResponse) {
       success: true,
     });
   }
+  // Warn that there was a wrong password attempt (date : wrong password, password tried, person's IP)
+  Consola.warn(
+    `${new Date().toLocaleString()} : Wrong password attempt, tried ${tried}, from ${
```

Comment:
``{tried}`` looks like a huge security risk to me. Imagine if the users enters the password incorrectly, but only by a single char. Then the password will be logged almost entirely and attackers just need to figure out a single character to get it.

I'd suggest to not log it at all or log only the length of the entered password.

Reply:
Yes but this log is on the serverside, it won't come up on the client. Also if the sever log is already compromised it's safe to assume the attacker has also access to environment variables